jupyterlab_git/handlers.py

```diff
@@ -473,7 +473,10 @@ async def post(self, path: str = ""):
         """
         data = self.get_json_body()
         commit_msg = data["commit_msg"]
-        body = await self.git.commit(commit_msg, self.url2localpath(path))
+        if data["amend"]:
+            body = await self.git.commit_ammend(self.url2localpath(path))
```

Comment:
Thinking a second time about it, it will be better to pass an additional argument `amend=True/False`. So we can handle in the future the edition of the commit message.